src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/_help.py

```diff
@@ -299,6 +299,18 @@
                 https://<deploy_user_name>@MyUniqueApp.scm.azurewebsites.net/MyUniqueApp.git
 """
 
+helps['webapp deployment source config-zip'] = """
+    type: command
+    short-summary: Perform deployment using the kudu zip push deployment. Supported on Webapps & function apps.
+    long-summary: By default, Kudu assumes that zip deployments do not require any build-related actions like npm install or dotnet publish.
+                  This can be overridden by including an .ini file on your zip file with the setting SCM_DO_BUILD_DURING_DEPLOYMENT = true to enable Kudu’s detection logic and build script generation process.
+                  Alternatley the setting can be enabled using the az webapp config appsettings set command.
+    examples:
+         - name: Perform deployment by using zip file content.
+          text: >
+            az webapp deployment source config-zip -g <myRG> -n <myAppName> --src "<zip file path location>"
+"""
```

Comment:
The indentation here is invalid.
When I run `az webapp deployment source config-zip --help` I get:
```
  File "/usr/local/lib/python2.7/dist-packages/yaml/parser.py", line 98, in check_event
    self.current_event = self.state()
  File "/usr/local/lib/python2.7/dist-packages/yaml/parser.py", line 393, in parse_block_sequence_entry
    "expected <block end>, but found %r" % token.id, token.start_mark)
ParserError: while parsing a block collection
  in "<string>", line 8, column 10:
             - name: Perform deployment by us ...
             ^
expected <block end>, but found '<block mapping start>'
  in "<string>", line 9, column 11:
              text: >
```